File: 351003/Naskevich/src/models/editor.py
from dataclasses import dataclass, field

from src.models.user_role import UserRole


@dataclass(kw_only=True)
class Editor:
    id: int = field(default=0, init=False)
    login: str
    password: str
    firstname: str
    lastname: str
    role: UserRole = UserRole.CUSTOMER

File: 351003/Gorodko/app/Publisher/Model/Editor.cs
using System.Text.Json.Serialization;

namespace Publisher.Model {

    public enum UserRole { ADMIN, CUSTOMER }

    public class Editor : BaseEntity {
        public string Login { get; set; } = string.Empty;
        public string Password { get; set; } = string.Empty;
        public string Firstname { get; set; } = string.Empty;
        public string Lastname { get; set; } = string.Empty;
        public string Role { get; set; } = "CUSTOMER";

        [JsonIgnore]
        [System.ComponentModel.DataAnnotations.Schema.NotMapped]
        public List<Tweet> Tweets { get; set; } = new();
    }

}